### DISCUSSION TITLE
Integrate with node.js (next.js)

### DISCUSSION THREAD

**Discussion by sasoryy94** [openai/whisper#543] [Q&A]
Hi everyone!

My question is, if it is posible to use whisper in a node.js environment. I have a project with Next.js and all the API calls i use them with next.js and i wanted to know if i can do the same with whisper because everything i found is in python.

In case that i can't yet and is it only in python, anyone knows how could i do an integration? basic whisper component with python and render it in next.js.

Many thanks!

**Comment by GoaGit** [comment]
@sasoryy94 check out this https://github.com/ggerganov/whisper.cpp/pull/260

**Comment by mayeaux** [comment]
I wrote a project in Node that uses Whisper you can check out my approach: https://github.com/mayeaux/generate-subtitles
